cGame_logic.cpp

```diff
@@ -255,7 +256,7 @@ void cGame::setMissionLost() {
     TIMER_shake = 0;
     mouse->setTile(MOUSE_NORMAL);
 
-    play_voice(SOUND_VOICE_08_ATR);
+    _soundplayer->playVoice(SOUND_VOICE_08_ATR, players[HUMAN].getHouse());
```

Comment:
I wonder if we should have a `player->playVoice` , which basically does this for us? A `cSoundPlayer` is ofcourse agnostic of which player, the player ID here is used to play the correct sound. In fact, this might be an abstraction to be introduced later? This ties into https://github.com/stefanhendriks/Dune-II---The-Maker/issues/186 so we can chose to defer it for now.